packages/material-ui/src/Select/Select.test.js

```diff
@@ -116,6 +116,7 @@ describe('<Select />', () => {
     const trigger = getByRole('button');
 
     fireEvent.mouseDown(trigger);
+    fireEvent.click(trigger);
```

Comment:
This is one indicator that this change is problematic. What happens if there is no click registered on the trigger due to interactions with other elements?

Reply:
There's no problem if the click occurs in other element, here it could even be in the `document.body`. I just need to issue it to clear the previous added listener or it will cause unexpected behavior in the next `fireEvent.mouseDown`  below.